doc/source/operator.rst

```diff
@@ -41,6 +41,25 @@ This will create the appropriate roles, service accounts and a deployment for th
    kube-system   dask-kubernetes-operator-775b8bbbd5-zdrf7   1/1     Running   0          74s
 
 
+Installing the operator with Helm
+---------------------------------
+
+Along with a set of kubernetes manifests, the operator has a basic Helm chart which can be used to manage the installation of the operator.
+The chart is published in the ``dask/helm-charts`` repository, and can be installed via:
```

Comment:
```suggestion
The chart is published in the `Dask Helm repo <https://helm.dask.org>`_ repository, and can be installed via:
```

I don't think folks will care too much about the specific of how the helm deployment works. Rather they will care about where to install it from.